### PR TITLE
UI: Unify actions empty state and "Learn more" with other views

### PR DESCRIPTION
**Pull request by johnnyaug**
image

### CODE REVIEW

File: webui/src/pages/repositories/repository/actions/index.jsx [hunk 1]
```diff
@@ -138,7 +125,7 @@ const ActionsList = ({ repo, after, onPaginate, branch, commit, onFilterBranch,
     if (!!error) content = <Error error={error}/>
 
     else if (loading) content = <Loading/>
-    else if (results.length === 0 && !nextPage) content = <GetStarted />
+    else if (results.length === 0 && !nextPage) content = <Alert variant="info" className={"mt-3"}>No actions have been logged yet.</Alert>
```
Comment (arielshaqed) [inline webui/src/pages/repositories/repository/actions/index.jsx:128]:
```suggestion
    else if (results.length === 0 && !nextPage) content = <Alert variant="info" className={"mt-3"}>No actions configured.</Alert>
```

(Most important to me here is not to say "logged", I find that verb confusing in this context.)

File: webui/src/pages/repositories/repository/actions/index.jsx [hunk 2]
```diff
@@ -174,8 +161,11 @@ const ActionsList = ({ repo, after, onPaginate, branch, commit, onFilterBranch,
                     <RefreshButton onClick={doRefresh}/>
                 </ActionGroup>
             </ActionsBar>
-
             {content}
+            <div>
+                {/* eslint-disable-next-line react/jsx-no-target-blank */}
+                Actions can be configured to run when predefined events occur. <a href="https://docs.lakefs.io/setup/hooks.html" target="_blank">Learn more.</a>
```
Comment (arielshaqed) [inline webui/src/pages/repositories/repository/actions/index.jsx:167]: This is intended _always_ to show up, even when there are actions?

File: webui/src/pages/repositories/repository/settings/branches.jsx [hunk 3]
```diff
@@ -38,8 +38,9 @@ const SettingsContainer = () => {
             </div>
             <div>
                 Define branch protection rules to prevent direct changes.
+                Changes to protected branches can only be done by merging from other branches.&nbsp;
```
Comment (arielshaqed) [inline webui/src/pages/repositories/repository/settings/branches.jsx:41]: Why do we need `&nbsp;`'?

**Review by arielshaqed — APPROVED** [review]
Great, thanks!

Looks much better.  Also anything that gets rid of an unneeded table for layout must be good :-)

File: webui/src/pages/repositories/repository/actions/index.jsx (hunk 2 above)
Comment (johnnyaug) [reply webui/src/pages/repositories/repository/actions/index.jsx:167]: Yes

**Pull request merged**